src/dotnet/ToolManifest/ToolManifestEditor.cs

```diff
@@ -359,10 +361,9 @@ private class SerializableLocalToolsManifest
 
             public string ToJson()
             {
-                var state = new JsonWriterState(options: new JsonWriterOptions { Indented = true });
```

Comment:
@ahsonkhan i need to get this in by the end of friday. Hope you can look at it soon. Thanks!